packages/protocol/engine/consensus/consensus.go

```diff
@@ -14,21 +16,25 @@ import (
 type Consensus struct {
 	Events *Events
 
-	*acceptance.Gadget
+	AcceptanceGadget        *acceptance.Gadget
```

Comment:
I think the name `AcceptanceGadget` and package `acceptance` is not fitting anymore. It is actually misleading. I'm not sure what we should call it but now it suggest that only acceptance is handled within but also block confirmation is part of it.

Reply:
:+1: I'll try to come up with something more general.